### ISSUE TITLE
SMS with code when new voice message or when I check the voicemail

### ISSUE THREAD

**Issue by Fazarel** [ubports/ubuntu-touch#1988]
- Device: observed with [Pixel 3a (sargo)](https://devices.ubuntu-touch.io/device/sargo/) and [Xiaomi Mi A2 (jasmine_sprout)](https://devices.ubuntu-touch.io/device/jasmine_sprout/)
- Channel: stable
- Build: OTA-22

### Steps to reproduce
Receive a voice message

### Expected behavior
A correctly readable SMS notification informing of the message

### Actual behavior
The SMS received displays code:

> //VVM:SYNC:ev=NM;txt=You have received a new message, call XXX;id=XXXXXXXXXX;c=0000X;t=v;s=+XXXXXXXXXX;dt=15/05/2022 17:02 +0200;l=33

And after calling the voicemail, another SMS:

> //VVM:SYNC:ev=MBU;c=00000

**Comment by lduboeuf** [comment]
I don't think UT supports VoiceMail protocol.
Meanwhile you can disable that feature from your provider.

**Comment by maciek134** [comment]
Looks simple enough, is there a documentation somewhere? Also which carrier is that?

**Comment by lduboeuf** [comment]
> Looks simple enough, is there a documentation somewhere? Also which carrier is that?

http://www.omtp.org/OMTP_VVM_Specification_v1_3_Final.pdf

that spec ? ^

**Comment by maciek134** [comment]
> that spec ? ^

Exactly, I guess I didn't know what to look for. I guess a partial implementation would be displaying these nicely / in the correct conversation (per the number that called you instead of the voicemail one)? And a proper one would have a menu for interacting with the voicemail box (which looking at the spec is just IMAP)
